### PR TITLE
remove-empty-directories: Add version 2.2

### PR DESCRIPTION
**Pull request by kunago**

### CODE REVIEW

File: bucket/remove-empty-directories.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "version": "2.2",
+    "description": "A tool to batch delete empty folders.",
+    "license": "GPL-3.0-or-later",
+    "homepage": "https://www.jonasjohn.de/red.htm",
+    "url": "https://downloads.sourceforge.net/project/rem-empty-dir/red-v2.2-portable.zip",
+    "hash": "f0114240d4cfc1f5bf81bdd8f38f6f161f15bd98e8ce2332123448f96ce7c423",
+    "bin": [
+      ["RED2.exe", "remove-empty-directories"]
```
Comment (Ash258) [inline bucket/remove-empty-directories.json:9]:
```suggestion
        "RED2.exe",
        [
            "RED2.exe",
            "remove-empty-directories"
        ]
```

File: bucket/remove-empty-directories.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    ],
+    "shortcuts": [
+      [
+        "RED2.exe",
+        "Remove Empty Directories"
+      ]
```
Comment (Ash258) [inline bucket/remove-empty-directories.json:15]:
```suggestion
        [
            "RED2.exe",
            "Remove Empty Directories"
        ]
```

File: bucket/remove-empty-directories.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+    "version": "2.2",
+    "description": "A tool to batch delete empty folders.",
+    "license": "GPL-3.0-or-later",
+    "homepage": "https://www.jonasjohn.de/red.htm",
+    "url": "https://downloads.sourceforge.net/project/rem-empty-dir/red-v2.2-portable.zip",
+    "hash": "f0114240d4cfc1f5bf81bdd8f38f6f161f15bd98e8ce2332123448f96ce7c423",
```
Comment (Ash258) [inline bucket/remove-empty-directories.json:7]:
```suggestion
    "hash": "sha1:00124046586afd9ca21957363b00da3c2e72cb9b",
```

File: bucket/remove-empty-directories.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{
+    "version": "2.2",
+    "description": "A tool to batch delete empty folders.",
+    "license": "GPL-3.0-or-later",
+    "homepage": "https://www.jonasjohn.de/red.htm",
```
Comment (Ash258) [inline bucket/remove-empty-directories.json:5]: image


```suggestion
    "description": "A tool to batch delete empty folders",
    "homepage": "https://www.jonasjohn.de/red.htm",
    "license": "LGPL-3.0-or-later",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]
How do you enable portable mode??

File: bucket/remove-empty-directories.json (hunk 3 above)
Comment (kunago) [reply bucket/remove-empty-directories.json:7]: Is SHA1 a standard to use?

Comment (Ash258) [reply bucket/remove-empty-directories.json:7]: It is automatic for sourceforge URLS...

**Comment by kunago** [comment]
Sorry, don't get this. I don't see anything to enable the portable mode. There is a portable version to download.

**Comment by Ash258** [comment]
Yes it is. But the configuration file is stored in APPDATA.

How do you enable the portable mode?

**Comment by kunago** [comment]
You can't, true. I looked at the code and obviously did not notice the config file in the APPDATA folder.

**Pull request closed without merging**